src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -30,7 +30,19 @@ void draw_settings_dialog(HostState &host) {
     ImGuiTabBarFlags settings_tab_flags = ImGuiTabBarFlags_None;
     ImGui::BeginTabBar("SettingsTabBar", settings_tab_flags);
 
+    // Core
+    if (ImGui::BeginTabItem("Core")) {
+        ImGui::PopStyleColor();
```

Comment:
The style you want should be pushed here (not the previous one popped) and popped at the end of the `if`.